packages/hardhat-plugin/src/index.ts

```diff
@@ -125,70 +125,28 @@ task("deploy")
         method: "eth_accounts",
       })) as string[];
 
-      try {
-        const deploymentId = givenDeploymentId ?? `network-${chainId}`;
-
-        const deploymentDir =
-          hre.network.name === "hardhat"
-            ? undefined
-            : path.join(hre.config.paths.ignition, "deployments", deploymentId);
-
-        const artifactResolver = new HardhatArtifactResolver(hre);
-
-        const result = await deploy({
-          config: hre.config.ignition,
-          provider: hre.network.provider,
-          artifactResolver,
-          deploymentDir,
-          ignitionModule: userModule,
-          deploymentParameters: parameters ?? {},
-          accounts,
-          verbose: logs,
-        });
+      const deploymentId = givenDeploymentId ?? `network-${chainId}`;
 
-        if (result.status === "success") {
-          console.log("Deployment complete");
-          console.log("");
-
-          for (const [
-            futureId,
-            { contractName, contractAddress },
-          ] of Object.entries(result.contracts)) {
-            console.log(`${contractName} (${futureId}) - ${contractAddress}`);
-          }
-        } else if (result.status === "failure") {
-          console.log("Deployment failed");
-          console.log("");
-
-          for (const [futureId, error] of Object.entries(result.errors)) {
-            const errorMessage =
-              "reason" in error ? (error.reason as string) : error.message;
-
-            console.log(`Future ${futureId} failed: ${errorMessage}`);
-          }
-        } else if (result.status === "timeout") {
-          console.log("Deployment halted due to timeout");
-          console.log("");
-
-          for (const { futureId, executionId, txHash } of result.timeouts) {
-            console.log(`  ${txHash} (${futureId}/${executionId})`);
-          }
-        } else if (result.status === "hold") {
-          console.log("Deployment held");
-        } else {
-          assertNeverResult(result);
-        }
-      } catch (err) {
-        // TODO: bring back cli ui
-        // if (DISPLAY_UI) {
-        //   // display of error or on hold is done
-        //   // based on state, thrown error display
-        //   // can be ignored
-        //   process.exit(1);
-        // } else {
-        throw err;
-        // }
-      }
+      const deploymentDir =
+        hre.network.name === "hardhat"
+          ? undefined
+          : path.join(hre.config.paths.ignition, "deployments", deploymentId);
+
+      const artifactResolver = new HardhatArtifactResolver(hre);
+
+      const result = await deploy({
+        config: hre.config.ignition,
+        provider: hre.network.provider,
+        artifactResolver,
+        deploymentDir,
+        ignitionModule: userModule,
+        deploymentParameters: parameters ?? {},
+        accounts,
+        verbose: logs,
+      });
+
+      // TODO: print a better result
```

Comment:
I removed this just to make it compile. We should reintroduce better messages.

Reply:
I think there is a rugged simplicity to stringified result.

Reply:
I have added a basic console.log print out of results leveraging the same helper message as is used in the ignition helper exception code.